gym/envs/classic_control/cartpole.py

```diff
@@ -13,17 +13,18 @@
 class CartPoleEnv(gym.Env):
     """
     Description:
-        A pole is attached by an un-actuated joint to a cart, which moves along a frictionless track. The pendulum starts upright, and the goal is to prevent it from falling over by increasing and reducing the cart's velocity.
+        A pole is attached by an un-actuated joint to a cart, which moves along a frictionless track. The pendulum
```

Comment:
what's the line wrap limit (max line length) here?

Reply:
Mine is set to 120 - just pushed to make this consistent.